components/style/properties/gecko.mako.rs

```diff
@@ -3077,19 +3078,25 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-lazy_static! {
-    static ref EMPTY_VARIABLES_STRUCT: nsStyleVariables = {
-        unsafe {
-            let mut variables: nsStyleVariables = unsafe { zeroed() };
-            Gecko_Construct_nsStyleVariables(&mut variables);
-            variables
-        }
-    };
-}
+// This is only accessed from the Gecko main thread.
+static mut EMPTY_VARIABLES_STRUCT: Option<nsStyleVariables> = None;
```

Comment:
I guess this works because nsStyleVariables doesn't have a Drop impl, only the wrapper struct.